scripts/release.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+# should be run as:
+# ./release.sh x.x.x start|end description
+set -eo pipefail
+
+USAGE="Usage:
+${0} x.x.x start|end description, E.g. to start a release run:
+${0} 2.4.0 start Maintenance release
+then to finish the release run:
+${0} 2.4.0 end"
+
+protocol_repo=valerifilipov/repo2
```

Comment:
let's make this environment variables with defaults:

```
PROTOTOL_REPO=${PROTOTOL_REPO:-aeternity/protocol}
```

Reply:
Addressed in f8d8974. Will change defaults in last commit